client/login/magic-login/handle-emailed-link-form-jetpack-connect.tsx

```diff
@@ -41,6 +43,9 @@ const HandleEmailedLinkFormJetpackConnect: FC< Props > = ( { emailAddress, token
 	const isExpired = useSelector(
 		( state ) => getMagicLoginCurrentView( state ) === LINK_EXPIRED_PAGE
 	);
+	const isWooCoreFlow = useSelector( isWoocommerceCoreProfilerFlow );
+	const isWcpayFlow = useSelector( isWooPaymentsFlow );
+	const isWooFlow = isWooCoreFlow || isWcpayFlow;
```

Comment:
Same here, I wonder if we can use  `wooDna.isWooDnaFlow()` here. For the core profiler flow, I think we should use `isWooPasswordlessJPCFlow` instead, which checks for both the Core Profiler and the new WooPayments JPC passwordless flow (still during testing).